tests/Behat/Tests/Definition/Pattern/PatternTransformerTest.php

```diff
@@ -66,12 +66,10 @@ public function testTransformPatternToRegexCacheAndRegisterNewPolicy()
         $this->assertEquals('/hello world/', $regex3);
     }
 
-    /**
-     * @expectedException \Behat\Behat\Definition\Exception\UnknownPatternException
-     * @expectedExceptionMessage Can not find policy for a pattern `hello world`.
-     */
     public function testTransformPatternToRegexNoMatch()
     {
+        $this->expectException('\Behat\Behat\Definition\Exception\UnknownPatternException');
+        $this->expectExceptionMessage("Can not find policy for a pattern `hello world`.");
```

Comment:
I'd put these just before the call in which we expect the exception instead, so that any code running before that cannot satisfy the expectation.